frame/system/src/offchain.rs

```diff
@@ -382,13 +382,14 @@ impl<T: SigningTypes> Clone for Account<T> where
 /// ```ignore
 ///	// im-online specific crypto
 /// type RuntimeAppPublic = ImOnline(sr25519::Public);
+///
 /// // wrapped "raw" crypto
 /// type GenericPublic = sr25519::Public;
 /// type GenericSignature = sr25519::Signature;
 ///
 /// // runtime-specific public key
-/// type Public = MultiSigner: From<sr25519::Public>;
-/// type Signature = MulitSignature: From<sr25519::Signature>;
+/// type GenericPublic = MultiSigner: From<sr25519::Public>;
```

Comment:
The multisigner is not a generic type but rather a concrete type that is used by the runtime as the comment suggests.